samples/v8-sample/README.md

```diff
@@ -1,3 +1 @@
-# V8 sample
-
-code taken for regression testing from https://github.com/v8/v8
+Code under src/ taken from https://github.com/v8/v8. Rights except redistribution reserved to the V8 project authors. See LICENSE.
```

Comment:
"Rights except redistribution reserved to the V8 project authors." is incorrect. It is BSD-3-Clause

Reply:
Yeah, I was mistaken. I also made that mistake on two others. I'll correct it soon